api/src/test/java/io/strimzi/test/StrimziExtension.java

```diff
@@ -710,6 +710,18 @@ private Statement installOperatorFromExamples(AnnotatedElement element, Statemen
                         LOGGER.info("STRIMZI_KAFKA_IMAGE_MAP={}", v);
                         ((ObjectNode) envVar).put("value", v);
                     }
+                    if (varName.matches("STRIMZI_KAFKA_CONNECT_IMAGE_MAP")) {
+                        String value = envVar.get("value").textValue();
+                        String v = TestUtils.changeOrgAndTagInImageMap(value);
+                        LOGGER.info("STRIMZI_KAFKA_CONNECT_IMAGE_MAP={}", v);
+                        ((ObjectNode) envVar).put("value", v);
+                    }
+                    if (varName.matches("STRIMZI_KAFKA_CONNECT_S2I_IMAGE_MAP")) {
+                        String value = envVar.get("value").textValue();
+                        String v = TestUtils.changeOrgAndTagInImageMap(value);
+                        LOGGER.info("STRIMZI_KAFKA_CONNECT_S2I_IMAGE_MAP={}", v);
+                        ((ObjectNode) envVar).put("value", v);
+                    }
```

Comment:
there is no need for the Kafka Mirror Maker image map `STRIMZI_KAFKA_MIRROR_MAKER_IMAGE_MAP` ?